udata/core/dataset/views.py

```diff
@@ -103,6 +104,19 @@ def get_context(self):
         return context
 
 
+@blueprint.route('/r/<uuid:id>', endpoint='resource')
```

Comment:
I renamed the property but I'd prefer to keep this pattern because it produce shorter URLs and I find it more coherent (I would have agree if patterns where resource related):
- `/datasets/some-dataset-slug/r/cc3cb0d0-87ed-4eb6-aced-470c083741cd`
- vs.
- `/datasets/some-dataset-slug/latest/cc3cb0d0-87ed-4eb6-aced-470c083741cd`

Are you OK with this ?